cli/src/test/java/org/opentosca/toscana/cli/ApiTest.java

```diff
@@ -0,0 +1,406 @@
+package org.opentosca.toscana.cli;
+
+import java.io.File;
+import java.io.IOException;
+
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import static org.junit.Assert.assertEquals;
+import static org.opentosca.toscana.cli.testStrings.SYSTEM_STATUS;
+
+@RunWith(JUnit4.class)
+public class ApiTest {
+
+    private ApiController api = null;
+    private MockWebServer server = null;
+    private MockResponse response = null;
+
+    @Before
+    public void setUp() throws IOException {
+        api = new ApiController(false, false);
+        server = new MockWebServer();
+        response = new MockResponse()
+            .addHeader("Content-Type", "application/json; charset=utf-8")
+            .addHeader("Cache-Control", "no-cache");
+    }
+
+    @After
+    public void tearDown() throws IOException {
+        server.shutdown();
+    }
+
+    @Test
+    public void testDeleteCsar() throws IOException {
+        assertEquals("Delete CSAR Archive: test.csar", api.deleteCsar("test.csar"));
+    }
+
+    @Test
+    public void testStatusTransformation() throws IOException {
+        assertEquals("Status of Transformation test", api.statusTransformation("test", "plat"));
+    }
+
+    @Test
+    public void testStopTransformation() throws IOException {
+        assertEquals("Transformation not running", api.stopTransformation("test", "plat"));
+    }
+
+    @Test
+    public void testShowTransformations() throws IOException {
+        assertEquals("All currently running Transformations: ", api.showTransformations());
+    }
+
+    @Test
+    public void testStatusPlatform() throws IOException {
+        assertEquals("Status of Platform ", api.statusPlatform("test"));
+    }
+
+    @Test
+    public void testShowPlatforms() throws IOException {
+        assertEquals("All currently available Platforms: ", api.showPlatforms());
```

Comment:
It is probably a good idea to put the "constants" in a centralized class

Reply:
Chris is right. If you reword the responses the test fails - this is unnecessary maintenance overhead